### PR TITLE
chore(GruntFile): check angularFiles (update)

### PR DESCRIPTION
**Pull request by petebacondarwin**

### CODE REVIEW

**Comment by petebacondarwin** [comment]
@matsko - can you take a look at these changes to your PR?

File: lib/grunt/validate-angular-files.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          if (!isHiddenFile && !detectedFiles[filePath]) {
+            grunt.log.error(filePath + " exists in the local file structure but isn't used by any module.");
+            errorsDetected = true;
+          }
+        }
+      });
+    });
+
+    if (errorsDetected) {
+      throw new Error("Not all files were properly detected in the local file structure.");
```
Comment (gkalpak) [inline lib/grunt/validate-angular-files.js]: This error message is not 100% accurate (as the errors might be due to files in the local file structure not being used).

**Comment by gkalpak** [comment]
LGTM (despite the inconsistent use of `'/"` :stuck_out_tongue:)

**Comment by matsko** [comment]
@petebacondarwin LGTM. Please upload as is. I will fix the error message that @gkalpak pointed out to something better in a follow-up commit.

**Pull request closed without merging**